.storybook/main.ts

```diff
@@ -1,7 +1,12 @@
 import type { StorybookConfig } from '@storybook/react-webpack5';
 
 const config: StorybookConfig = {
-  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-mdx-gfm'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-mdx-gfm',
+    '@storybook/addon-viewport',
```

Comment:
Storybook has no documentation showing this addon is needed to get the viewport toolbar working. However, when you strip out all logic from the preview.tsx and main.ts files (e.g., emulating a bare bones project), even the default viewports don't work. Only way I could get it working was adding this addon.